DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json

```diff
@@ -1,5 +0,0 @@
-﻿{
```

Comment:
Can't delete this, it's used by the pipeline to populate connection strings during deployment.